EIPS/eip-1185.md

```diff
@@ -0,0 +1,80 @@
+---
+eip: 1185
+title: Storage of DNS Records in ENS
+author: Jim McDonald <Jim@mcdee.net>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2018-06-26
+requires: 137
+discussions-to: https://github.com/ethereum/EIPs/pull/1185
+---
+  
+## Abstract
+This EIP defines a resolver profile for ENS that provides features for storage and lookup of DNS records. This allows ENS to be used as a store of authoritative DNS information.
+
+## Motivation
+ENS is a highly desirable store for DNS information.  It provides the distributed authority of DNS without conflating ownership and authoritative serving of information.  With ENS, the owner of a domain has full control over their own DNS records.  Also, ENS has the ability (through smart contracts) for a domain's subdomains to be irrevocably assigned to another entity.
+
+## Specification
+
+The resolver profile to support DNS on ENS builds on the standalone resolver as defined in #137.
```

Comment:
What's "the standalone resolver"?

Reply:
This is the name used at https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md#standalone-resolver

Reply:
That's just one implementation of a resolver; resolver profiles are independent of implementations.

Reply:
I'll change it to say that it follows the resolver specification as per #137 instead.